records.py

```diff
@@ -237,8 +236,10 @@ def __init__(self, db_url=None, **kwargs):
         if not self.db_url:
             raise ValueError('You must provide a db_url.')
 
+        self._engine = create_engine(self.db_url, **kwargs)
```

Comment:
@RazerM here is the issue that relates to this: https://github.com/kennethreitz/records/issues/67